comments.md

```diff
@@ -0,0 +1,44 @@
+# New Comments
+-(ZH) I'm now thinking that when you are "Building" resources, you should be using only a build definition object.  Then once, it is deployed, it would return a PowerFormResource that contains your build defnition as well as the cloud definitions.  Thoughts on this?
+
+# Comments
+-liking the options input into new-definition; this looking solid 
+   (ZH) me too
+
+-Context should be able to take a Name also, arbitrary but we will key off of it for top level of state
+   (ZH) I am not sure why we would need names for contexts.  My thinking is that it is only needed for deployment.  The context sets properties on the resource itself.  So after the resource is built from context, you can change those properties.  Maybe we are thinking of going two different directions here?
+
```

Comment:
i guess in back of my mind i see possibility of a dc to be a thing into itself.  If you had a UI you could drill into the various DC's as top level entry point, which always ties back to some definition script(s) that are sitting somewhere in your repo.   An example DC could be a 'product' deploy script that has a sub and multiple RG's, and we say this is the 'portal deploy'

Reply:
yeah I think we will have to talk/demo this more.  If that's the case we need some sort of backend to store that info in if we are talking a UI.  And while we can think about that scenario, I am not sure it makes sense to build that out without a having a backend arch first..